### DISCUSSION TITLE
2008/10/23/%E5%8D%97%E4%BA%AC%E6%8B%9B%E8%81%98%E4%B9%8B%E6%97%85

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#247] [Announcements]
# 2008/10/23/%E5%8D%97%E4%BA%AC%E6%8B%9B%E8%81%98%E4%B9%8B%E6%97%85

动车

https://blog.qizhen.xyz/2008/10/23/%E5%8D%97%E4%BA%AC%E6%8B%9B%E8%81%98%E4%B9%8B%E6%97%85

**Comment by ruanqizhen** [comment]
之前博客下的留言：


2008年11月29日 21:57 编辑
我从法兰克福坐ice到柏林，4个半小时，大部分时间都在150左右，只有快到柏林一段才上200。本来慕尼黑说建磁悬浮，后来因为资金和所谓的环境的问题而不是技术的问题取消了。

回复
hongfei
2008年10月25日 20:35 编辑
等着慢慢好起来吧

回复

李琦
2011年02月20日 17:08 编辑
阮先生，很高兴看到您这类的关于招聘的文章，对于今年要找工作的我很有帮助，我也要去拍张像样的照片

回复
Pingback： 生活随笔 « 凡人琐忆 编辑
文婷
2009年01月1日 17:07 编辑
我看像诺贝尔得奖者

回复
yoyo
2008年11月22日 19:40 编辑
挺神奇

回复
hongzhi
2008年10月27日 18:40 编辑
你出落的愈发标致了，像个科学家，呵呵。

回复
Jing
2008年10月26日 19:55 编辑
Nice written piece. 如果我是招聘者，确实也会这么想

回复
竹贤
2008年10月26日 07:02 编辑
眼不在大，有神就行。

回复
ruirui
2008年10月25日 22:43 编辑
阅毕.
没有真人帅.

回复
hongfei
2008年10月25日 10:15 编辑
看看你拍的神奇照片是什么样子？期待ing

回复
ruirui
2008年10月25日 07:19 编辑
这么说来…我终于明白为何我当年几乎拿到所有的面试,无论筛选比例多么离奇地苛刻….
btw,你有很多拍得不错的照片，做个NI形象代言人足够了。

回复
xia
2008年10月24日 22:22 编辑
marketing marketing marketing ….我要marketing的工作。。。：‘(

回复
雪狼
2008年10月24日 21:34 编辑
人力资源管理是我以前就喜欢的职业

回复
arrowpig
2008年10月24日 17:43 编辑
还是我帮你拍得那张在青岛钓鱼的照片最好了

回复

Peter
2008年11月18日 21:48 编辑
前几天刚刚电话面试，感觉用英语沟通会有压迫感！
猎头说，要不要再来一次？
我心里却想“杀了我得了！”

回复
hongfei
2008年10月27日 19:50 编辑
呼呼，找工作不易，原来给工作也不容易。
庆幸自己的工作已经定了。

回复

hongfei
2008年11月3日 20:44 编辑
应当让更多的应届生看看

回复
ilin
2008年11月3日 15:11 编辑
这样的选拔很可怕……

回复

hongfei
2008年11月3日 20:48 编辑
招聘和被招聘的都不容易啊。
 

回复
ilin
2008年11月3日 15:07 编辑
可以知道你们是什么公司么？很想知道国内目前公司的招聘动态和要求，呵呵～

回复

hongfei
2008年11月23日 10:40 编辑
(*^__^*) 嘻嘻……，看来我是比较幸运的，什么时候都可以睡着。时差基本2天就搞定了。
看来你是个善于思考的人，喔喔

回复
listen
2008年11月5日 22:49 编辑
握个手，我也讨厌做飞机，长途旅行如果时间够充分，我宁愿选择火车，看窗口的风景也是享受旅途的过程之一。

回复
